Resources/doc/index.rst

```diff
@@ -350,3 +350,8 @@ class (as shown above), you can access it in the ``load()`` method.
 
 .. _DoctrineFixturesBundle: https://github.com/doctrine/DoctrineFixturesBundle
 .. _`Doctrine Data Fixtures`: https://github.com/doctrine/data-fixtures
+
+Hautelook/AliceBundle
+---------------------
+
+You can also use the popular [hautelook/AliceBundle](https://github.com/hautelook/AliceBundle) bundle to manage your fixtures with [nelmio/alice](https://github.com/nelmio/alice) and [fzaninotto/Faker](https://github.com/fzaninotto/Faker).
```

Comment:
to be consistent, you should make these links with footnotes as well (see DoctrineFixturesBundle in this file for an example).

i would not assume that people know what alice and faker are. i would do a title like "Automatically Generated Fixtures" and quickly explain (not a large doc with example, just a sentence or two for people who might not know what it is) what faker and alice are, then say that you can use hautelook/alice-bundle to use them with doctrine fixtures.